data/transactions/logic/opcodes.go

```diff
@@ -563,7 +565,7 @@ var OpSpecs = []OpSpec{
 	{0x8b, "frame_dig", opFrameDig, proto(":a"), fpVersion, immKinded(immInt8, "i").typed(typeFrameDig)},
 	{0x8c, "frame_bury", opFrameBury, proto("a:"), fpVersion, immKinded(immInt8, "i").typed(typeFrameBury)},
 	{0x8d, "switch", opSwitch, proto("i:"), 8, detSwitch()},
-	// 0x8e will likely be a switch on pairs of values/targets, called `match`
+	{0x8e, "match", opMatch, proto(":", "[N items]", ""), 8, detSwitch().trust()},
```

Comment:
Here's where you need something like, "[N items], A".

Reply:
Actually, how about using:
`"[T1 T2 T3 ... TN], B"`
That way you have names for the targets (which I renamed T for target to prevent confusion because we normally named the arguments A and B with A _deeper_ in the stack.

That should make your documentation of the behavior easier.

Reply:
oops, they are not targets, they are match values.  So I'd be ok with `"[A1 A2 A3 ... An], B"`